### PR TITLE
[docs] pipeline docs for latte

### PR DESCRIPTION
**Pull request by a-r-r-o-w**
# What does this PR do?

It turns out we missed adding the pipeline docs in the Latte PR, and only added it for the Latte Transformer 3D model. @maxin-cn Let me know if you want anything else to be added.

Maybe a line about #8842 (ff-chunked inference) could be added once that's merged.

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@yiyixuxu

### CODE REVIEW

**Comment by maxin-cn** [comment]
> # What does this PR do?
> It turns out we missed adding the pipeline docs in the Latte PR, and only added it for the Latte Transformer 3D model. @maxin-cn Let me know if you want anything else to be added.
> 
> Maybe a line about #8842 (ff-chunked inference) could be added once that's merged.
> 
> ## Who can review?
> Anyone in the community is free to review the PR once the tests have passed. Feel free to tag members/contributors who may be interested in your PR.
> 
> @yiyixuxu

Thank you for continuing to refine the inference process of Latte. I don't have anything extra to add at the moment.

File: docs/source/en/api/pipelines/latte.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+**Highlights**: Latte is a latent diffusion transformer proposed as a backbone for modeling different modalities (trained for text-to-video generation here). It achieves state-of-the-art performance across four standard video benchmarks.
```
Comment (sayakpaul) [inline docs/source/en/api/pipelines/latte.md:25]: Can we hyperlink the benchmarks section?

File: docs/source/en/api/pipelines/latte.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+<!-- # Copyright 2024 the Latte Team and The HuggingFace Team. All rights reserved.
```
Comment (sayakpaul) [inline docs/source/en/api/pipelines/latte.md:1]: We usually only include Hugging Face for documentation.

**Review by sayakpaul — APPROVED** [review]
Some minor comments.

File: docs/source/en/api/pipelines/latte.md (hunk 1 above)
Comment (maxin-cn) [reply docs/source/en/api/pipelines/latte.md:25]: Here I provide the hyperlink for the four benchmarks (https://github.com/Vchitect/Latte/blob/main/docs/datasets_evaluation.md#download-datasets).

Comment (a-r-r-o-w) [reply docs/source/en/api/pipelines/latte.md:25]: Do you mean the hyperlink to arxiv benchmarks section here?

File: docs/source/en/api/pipelines/latte.md (hunk 2 above)
Comment (maxin-cn) [reply docs/source/en/api/pipelines/latte.md:1]: Okay. We can delete the Latte team.

**Comment by a-r-r-o-w** [comment]
@sayakpaul Requesting another review before merge for latest changes

**Review by sayakpaul — APPROVED** [review]

**Pull request merged**